app/services/messages/operations.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+module Messages
```

Comment:
Option, you could also make a rails concern out of this. And nest it under the model

We've done this for a few things: e.g https://github.com/crossroads/api.goodcity/blob/master/app/models/concerns/operations/designation_operations.rb